### PR TITLE
First release version

### PR DESCRIPTION
**Pull request by vreuter**
- [x] `docs/changelog.md`
- [x] `divvy/_version.py`

### CODE REVIEW

File: docs/autodoc_build/divvy.md [hunk 1]
```diff
@@ -174,7 +166,7 @@ D.pop(k[,d]) -> v, remove specified key and return the corresponding value.
 
 If key is not found, d is returned if given, otherwise KeyError is raised.
 ```python
-def pop(self, key, default=<object object at 0x7f5d0764d030>):
+def pop(self, key, default=<object object at 0x7f2dffda3030>):
 ```
```
Comment (nsheff) [inline docs/autodoc_build/divvy.md]: what is this object notation? it's new to me

Comment (vreuter) [reply docs/autodoc_build/divvy.md]: No idea...this is inherited from `MutableMapping` I think

**Comment by nsheff** [comment]
Yes. 
Travis is also failing (I guess because there are no tests run).

**Comment by michalstolarczyk** [comment]
> [ ] [PyPI Release](https://github.com/databio/divvy/issues/13)

It looks like the release is the only thing that's left to do

**Comment by vreuter** [comment]
Sounds great. @nsheff is this good to merge? I listed the release here but think we should merge, tag "release" here on GH, then PyPI.

**Comment by vreuter** [comment]
Wait for https://github.com/pepkit/divvy/pull/20

**Comment by nsheff** [comment]
Ok I'm doing my final check and planning to release now.

File: divvy/compute.py [hunk 2]
```diff
@@ -262,6 +262,7 @@ def write_script(self, output_path, extra_vars=None):
         if extra_vars:
             if not isinstance(extra_vars, list):
                 extra_vars = [extra_vars]
+            extra_vars.reverse()
```
Comment (vreuter) [inline divvy/compute.py]: Do we want the input list to actually be changed? I think the reversal will be visible after this function call.

Comment (nsheff) [reply divvy/compute.py]: I was imagining they'd just be giving a list on-the-fly...but yeah, that could be a problem if they're giving a named list...

**Pull request merged**